lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -4086,3 +4083,30 @@ void mme_app_handle_modify_bearer_rsp(
   }
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
+
+void send_s11_modify_bearer_request(
+    ue_mm_context_t* ue_context_p, pdn_context_t* pdn_context_p,
+    MessageDef* message_p) {
+  OAILOG_FUNC_IN(LOG_MME_APP);
+  Imsi_t imsi = {0};
+  IMSI64_TO_STRING(
+      ue_context_p->emm_context._imsi64, (char*) (&imsi.digit),
+      ue_context_p->emm_context._imsi.length);
+
+  if (pdn_context_p->route_s11_messages_to_s8_task) {
+    OAILOG_INFO_UE(
```

Comment:
debug level?

Reply:
My understanding on info level would capture the call flow; So I had logged with info, to know where s11 messages would be routed through.
Let me know your thoughts on same